<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogJoinGame</class>
 <widget class="QDialog" name="DialogJoinGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="errorMsg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(237, 51, 59)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de partida:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_gameName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="infoClientConnected">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(51, 209, 122)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectarse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
